src/com/trainee/aizaz/selenium/AddCartSelenium.java

```diff
@@ -0,0 +1,101 @@
+package com.trainee.aizaz.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * The purpose of this script is to Launch APP > Verify LOGO > Verify Page Title >
+ * Click on T shirts > Add item > Grab the Total Price >
+ * Click on checkout > Verify the Total Price is same as it is in cart
+ * @author aizaz
+ */
+
+public class AddCartSelenium extends Thread {
+    public static void main(String[] args) throws InterruptedException {
+        String chromeDriverPath = System.getProperty("user.dir") + "/drivers/chromedriver";
+        System.out.println("ChromeDrive located At Path: " + chromeDriverPath);
+        System.setProperty("webdriver.chrome.driver", chromeDriverPath);
+        WebDriver driver = new ChromeDriver();
+        /**
+         * implicit wait provide default waiting time between
+         * each consecutive command across the entire test script.
+         */
+        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        driver.get("http://automationpractice.com/index.php");
+        /**
+         * To find whether logo is displayed or not we need to go for isDisplayed() method
+         * which returns boolean value
+         */
+        boolean logo = driver.findElement(By.xpath("//img[@class='logo img-responsive']")).isDisplayed();
+        /**
+         * this logic displays whether the logo is displayed or not
+         */
+        if (logo == true) {
+            System.out.println("LOGO DISPLAYED SUCESSFULLY");
+
+        } else {
+            System.out.println("LOGO NOT DISPLAYED");
+        }
```

Comment:
This is not best way to compare. you already have boolean. This type of comparison  we should do with Strings

if(logo) --> is enough